tests/src/test/java/com/netflix/msl/msg/MessageBuilderSuite.java

```diff
@@ -1151,25 +1153,25 @@ public void setPeerUser() throws MslMessageException, MslEncodingException, MslC
         
         @Test(expected = MslInternalException.class)
         public void setPeerUserNoPeerMasterToken() throws MslEncodingException, MslCryptoException, MslException {
-            final MessageBuilder builder = MessageBuilder.createRequest(p2pCtx, null, null);
+            final MessageBuilder builder = messageFactory.createRequest(p2pCtx, null, null);
             builder.setUser(PEER_USER_ID_TOKEN.getUser());
         }
         
         @Test(expected = MslInternalException.class)
         public void setPeerUserHasPeerUserIdToken() throws MslEncodingException, MslCryptoException, MslException {
-            final MessageBuilder builder = MessageBuilder.createRequest(p2pCtx, null, null);
+            final MessageBuilder builder = messageFactory.createRequest(p2pCtx, null, null);
             builder.setPeerAuthTokens(PEER_MASTER_TOKEN, PEER_USER_ID_TOKEN);
             builder.setUser(USER_ID_TOKEN.getUser());
         }
         
         @Test(expected = MslInternalException.class)
         public void negativeMessageId() throws MslException {
-            MessageBuilder.createRequest(trustedNetCtx, null, null, -1);
+            MessageBuilder bldr = messageFactory.createRequest(trustedNetCtx, null, null, -1);
```

Comment:
You don't need an lvalue. Adding one will just generate warnings about an unused variable. Same in `tooLargeMessageId()` and `negativeMessageId()`.

Reply:
done.